orbit/pkg/table/dscl/dscl_darwin.go

```diff
@@ -69,7 +69,7 @@ func Generate(ctx context.Context, queryContext table.QueryContext) ([]map[strin
 		return nil, errors.New("missing key argument")
 	}
 
-	cmd := exec.Command("dscl", ".", "-"+command, path, key)
+	cmd := exec.Command("/usr/bin/dscl", ".", command, path, key)
```

Comment:
This is the kind of thing we would like to catch during development:
```
orbit/pkg/table/dscl/dscl_darwin.go:72:9: G204: Subprocess launched with a potential tainted input or cmd arguments (gosec)
        cmd := exec.Command("dscl", ".", "-"+command, path, key)
```

In this specific case there's no issue because command can only be `read` (dscl handles both `-read` and `read`), and `path` and `key` are handled by `exec.Command`.